src/main/java/com/nedap/archie/rules/evaluation/evaluators/FunctionUtil.java

```diff
@@ -0,0 +1,70 @@
+package com.nedap.archie.rules.evaluation.evaluators;
+
+import com.google.common.collect.Lists;
+import com.nedap.archie.rules.evaluation.FunctionCallException;
+import com.nedap.archie.rules.evaluation.Value;
+import com.nedap.archie.rules.evaluation.ValueList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by pieter.bos on 07/04/2017.
+ */
+public class FunctionUtil {
+
+    public static double castToDouble(Value value) throws FunctionCallException {
+        if(value.getValue() instanceof Long) {
+            return ((Long) value.getValue()).doubleValue();
+        } else if (value.getValue() instanceof Double) {
+            return (double) value.getValue();
+        }
+        throw new FunctionCallException("cannot cast " + value.getValue().getClass() + " to a number");
+    }
+
+    public static int checkLength(List<ValueList> arguments) {
```

Comment:
Had to look up the usage to understand this function, maybe rename to checkEqualLength ?